src/config.cpp

```diff
@@ -216,6 +311,50 @@ std::set<std::string> listPopulations(const std::vector<SubnetworkFiles>& networ
                    [](std::pair<std::string, SubnetworkFiles> p) { return p.first; });
     return result;
 }
+
+template <typename Population>
+void checkDuplicatePopulationNames(const std::vector<SubnetworkFiles>& networkNodes) {
+    using PopulationStorage = bbp::sonata::PopulationStorage<Population>;
+    std::set<std::string> check;
+
+    for (const auto& network : networkNodes) {
+        const auto population = PopulationStorage(network.elements, network.types);
+        for (auto name : population.populationNames()) {
+            if(check.find(name) != check.end())
+                throw SonataError(fmt::format("Duplicate population name '{}'", name));
+            check.insert(name);
+        }
+    }
+}
+
+void checkBiophysicalNodePopulations(
+        const std::vector<SubnetworkFiles>& networkFiles,
+        const std::map<std::string, PopulationProperties>& nodePopulations) {
+
+    using PopulationStorage = bbp::sonata::PopulationStorage<bbp::sonata::NodePopulation>;
```

Comment:
`NodeStorage` exists in `nodes.h`

Reply:
Removed `using` statement and replaced with `NodeStorage`, thanks for the tip!